lessons/11_Lesson11_vortexSourcePanelMethod.ipynb

```diff
@@ -151,26 +154,15 @@
    ],
    "source": [
     "# plot geometry\n",
-    "\n",
-    "# parameters to have a nice plot\n",
-    "val_x, val_y = 0.1, 0.2              \n",
-    "\n",
-    "xp_min, xp_max = x.min(), x.max()    \n",
-    "yp_min, yp_max = y.min(), y.max()\n",
-    "\n",
-    "# plot limits\n",
-    "xp_start, xp_end = xp_min-val_x*(xp_max-xp_min), xp_max+val_x*(xp_max-xp_min)\n",
-    "yp_start, yp_end = yp_min-val_y*(yp_max-yp_min), yp_max+val_y*(yp_max-yp_min)\n",
-    "\n",
-    "# plot\n",
-    "size = 10\n",
-    "pyplot.figure(figsize=(size, (yp_end-yp_start)/(xp_end-xp_start)*size))\n",
-    "pyplot.grid(True)\n",
+    "width = 10\n",
+    "pyplot.figure(figsize=(width, width))\n",
+    "pyplot.grid()\n",
     "pyplot.xlabel('x', fontsize=16)\n",
     "pyplot.ylabel('y', fontsize=16)\n",
-    "pyplot.xlim(xp_start, xp_end)\n",
-    "pyplot.ylim(yp_start, yp_end)\n",
-    "pyplot.plot(x, y, color='k', linestyle='-', linewidth=2);"
+    "pyplot.plot(x, y, color='k', linestyle='-', linewidth=2)\n",
+    "pyplot.xlim(-0.1, 1.1)\n",
+    "pyplot.ylim(-0.1, 0.1)\n",
+    "pyplot.gca().set_aspect('equal', adjustable='box');"
```

Comment:
I think it's better to avoid using the `gca` handler whenever possible. This should give the same effect:
```python
pyplot.axis('scaled', adjustable='box')
pyplot.xlim(-0.1, 1.1)
pyplot.ylim(-0.1, 0.1)
```

although note that you need to put the axis limits _after_ setting the aspect ratio